sechub-schedule/src/main/java/com/daimler/sechub/domain/schedule/access/ScheduleAccessRepository.java

```diff
@@ -11,9 +11,12 @@
 
 public interface ScheduleAccessRepository extends JpaRepository<ScheduleAccess, ProjectAccessCompositeKey> {
 
+	@Query(value="SELECT EXISTS(SELECT * FROM "+TABLE_NAME+" where "+COLUMN_PROJECT_ID+" = ?1)",nativeQuery=true)
+	public Boolean hasProjectUserAccess(String projectId);
```

Comment:
Please try to use simple data types where possible. (of course not when using BigInteger etc.) 

Reason: Boolean can be `null` - so having methods which returns a "Boolean" instead of a simple "boolean" do implicit bring up the question at caller side : can this be null? When callers do NOT think about the call and just do an
```if (isXYZExampleForBoolean()) ``` this does autoboxing and when Boolean result is null, then you got an unexpected NPE. 

Yes, in this case this should never happen.
I do not know if hibernate/JPA does internally use a boolean and you got boolean->Boolean->boolean conversion, but you always do a Boolean->boolean autoboxing, so the result can be defined as simple data type.

Reply:
done